docs/program/apis/cloud.md

```diff
@@ -188,9 +188,281 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
+
+- `GRPCError`: This error is raised if either an invalid name (such as "") or invalid parent location ID is passed.
+
+**Returns:**
+
+- [(viam.proto.app.Location)](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Location): The newly created location.
+
+```python {class="line-numbers linkable-line-numbers"}
+my_new_location = await cloud.create_location(name="Robotville", parent_location_id="111ab12345")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.create_location).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetLocation
+
+Get a {{< glossary_tooltip term_id="location" text="location" >}} by its location ID.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `location_id` [(Optional[string])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the location to get.
+Defaults to the location ID provided at `AppClient` instantiation.
+
+**Raises:**
+
+- `GRPCError`: This error is raised if an invalid location ID is passed, or if one isn't passed and no location ID was provided at `AppClient` instantiation.
+
+**Returns:**
+
+- [(viam.proto.app.Location)](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Location): The location corresponding to the provided ID.
+
+```python {class="line-numbers linkable-line-numbers"}
+location = await cloud.get_location(location_id="123ab12345")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.get_location).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### UpdateLocation
+
+Change the name of a {{< glossary_tooltip term_id="location" text="parent location" >}} and/or assign it a new location.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `location_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the location to update.
+- `name` [Optional(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Optional new name to update the location name to.
+If nothing is passed, the name is not changed.
+- `parent_location_id` [Optional(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Optional ID of the new location to move the location to.
+If nothing is passed, the location does not move.
+If an empty string is passed, the location moves up to the top level.
+
+**Raises:**
+
+- `GRPCError`: This error is raised if an invalid location ID is passed, or if one isn't passed and no location ID was provided at `AppClient` instantiation.
+
+**Returns:**
+
+- [(viam.proto.app.Location)](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Location): The newly updated location.
+
+```python {class="line-numbers linkable-line-numbers"}
+# The following line takes the location with ID "abc12abcde" and moves it
```

Comment:
```suggestion
# The following line takes the location with ID "abc12abcde" and moves it to be a sub-location of the location with ID "xyz34xxxxx"
```
(opt) Would put this all on one line myself bc I personally think it looks better, but highly highly optional

Reply:
I think I'll leave as-is because on narrow screens it will become quite wide. However, maybe I should split the parameters onto multiple lines as well now that I look at it...

Reply:
(done splitting BTW)